src/NServiceBus.AcceptanceTests/Audit/When_auditing_message_with_TimeToBeReceived.cs

```diff
@@ -13,7 +13,7 @@ public class When_auditing_message_with_TimeToBeReceived : NServiceBusAcceptance
         public async Task Should_not_honor_TimeToBeReceived_for_audit_message()
         {
             var context = await Scenario.Define<Context>()
-            .WithEndpoint<EndpointWithAuditOn>(b => b.Given(bus => bus.SendLocalAsync(new MessageToBeAudited())))
+            .WithEndpoint<EndpointWithAuditOn>(b => b.When(bus => bus.SendLocalAsync(new MessageToBeAudited())))
```

Comment:
`When` doesn't read so good without a condition. Should we consider splitting into

`WhenStarted(bus=>)` and `When(condition, bus=>)`  ?